lib/utils/RedisCommander.ts

```diff
@@ -8579,6 +8579,45 @@ interface RedisCommander<Context extends ClientContext = { type: "default" }> {
     justid: "JUSTID",
     callback?: Callback<unknown[]>
   ): Result<unknown[], Context>;
+  
+  xautoclaimBuffer(
```

Comment:
It is simple copy-paste for non-buffer version typings - I assume it is okay, but let me know if it doesn't...